pages/about.tsx

```diff
@@ -369,24 +368,24 @@ const AboutPage = () => {
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H3>Access Sets</H3>} anchorKey="accesssets">
+      <SectionWrapper header={<H3>Access List</H3>} anchorKey="access_list">
         <p className="pb-6">
-          Access sets are defined per external transaction, and not per call.
+          Access list is defined per external transaction, and not per call.
```

Comment:
Any reason why you updated wording in this file? Pretty out of scope of this PR.